src/algos/attack_bad_weights.py

```diff
@@ -0,0 +1,35 @@
+from collections import OrderedDict
+from torch import Tensor
+from typing import TypeAlias, Dict, List, Union, Tuple, Optional
+import random
+
+ConfigType: TypeAlias = Dict[
```

Comment:
Can you create a separate file `types.py` that stores the ConfigType? And we should just import from there in the future.